File: 27-JRCircleDemo/27-JRCircleDemo/ViewController.m
//
//  ViewController.m
//  27-JRCircleDemo
//
//  Created by jackfrow on 2019/11/29.
//  Copyright © 2019 jackfrow. All rights reserved.
//

#import "ViewController.h"
#import "JRCircleView.h"
#import "UIColor+Hex.h"
@import Masonry;

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    self.view.backgroundColor = [UIColor colorWithHexString:@"110918"];

    JRCircleView* circle = [[JRCircleView alloc] init];
    [self.view addSubview:circle];
    
    [circle mas_makeConstraints:^(MASConstraintMaker *make) {
        make.center.mas_equalTo(self.view);
    }];
    
}


@end

File: 51.SDragDemo/SDragDemo/ViewController.swift
//
//  ViewController.swift
//  SDragDemo
//
//  Created by jackfrow on 2023/4/3.
//

import UIKit


class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        
        
        let dgv = SGDragView(frame: CGRect(x: 0, y: UIScreen.main.bounds.height - 400 + 250, width:UIScreen.main.bounds.width , height: 400))
        dgv.backgroundColor = UIColor.green
        
        self.view.addSubview(dgv)
        
    }


}

